cmd/telliot/main.go

```diff
@@ -222,22 +227,20 @@ func balanceCmd(cmd *cli.Cmd) {
 	cmd.VarArg("ADDRESS", &addr, "ethereum public address")
 	cmd.Spec = "[ADDRESS]"
 	cmd.Action = func() {
-		// Using values from context, until we have a function that setups the client and returns as values, not as part of the context
-		commonAddress := ctx.Value(tellorCommon.PublicAddress).(common.Address)
 		var zero [20]byte
 		if bytes.Equal(addr.addr.Bytes(), zero[:]) {
-			addr.addr = commonAddress
+			addr.addr = acc.Address
```

Comment:
I really hope we can get rid of these global variables with the cli refactoring. the final goal is to use only variables that are passed to a function. Just mentioning it here to make sure we are on the same page.

Reply:
Yes, they'll be gone!